<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0JT4R6G" name="Regression Test" tests="3" failures="1" timestamp="29 Aug 2021 05:45:02 GMT" time="107.841" errors="0">
  <testcase name="feature" time="34.616" classname="com.tms.testRunner.TestRunner"/>
  <testcase name="feature" time="44.131" classname="com.tms.testRunner.TestRunner"/>
  <testcase name="feature" time="28.808" classname="com.tms.testRunner.TestRunner">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: expected [true] but found [false]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: expected [true] but found [false]
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at com.tms.testRunner.TestRunner.feature(TestRunner.java:38)
Caused by: java.lang.AssertionError: expected [true] but found [false]
	at com.tms.stepdefinitions.SettingsPageStepDefinitions.Success_message_display(SettingsPageStepDefinitions.java:106)
	at âœ½.Then I should see a popup having text as Mail server has been NOT connected successfully.(src/test/resources/features/Settings/emailSetup.feature:21)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Regression Test -->
